DuckDuckGo/AppTPAboutView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AppTPAboutView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+#if APP_TRACKING_PROTECTION
+
+struct AppTPAboutView: View {
+    var aboutBody: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Const.Size.stackSpacing) {
+                Text(UserText.appTPAboutTitle)
+                    .font(Font(uiFont: Const.Font.titleFont))
+                
+                Group {
+                    Text(UserText.appTPAboutContent1)
+                    + Text(UserText.appTPAboutContent2)
+                        .fontWeight(.semibold)
+                    + Text(UserText.appTPAboutContent3)
+                }
```

Comment:
Since this `Group` is technically just containing a single view (three `Text` views concatenated), is it redundant? I tested it just now in a preview and the view appeared the same after removing the group.